src/Http/Http.Extensions/test/Microsoft.AspNetCore.Http.Extensions.Tests.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="FSharp.Core" />
```

Comment:
We don’t need to reference F# here directly. Take a look at the MVC logic

Reply:
Even in the test project? `Shared.Tests.csproj` (which contains the existing `ObjectMethodExecutor` tests) has a reference like this:

https://github.com/dotnet/aspnetcore/blob/72b0269372a6be709642b6c36f6aacc820ccfe6e/src/Shared/test/Shared.Tests/Microsoft.AspNetCore.Shared.Tests.csproj#L42

Reply:
The test project is fine.

Reply:
Ah I read this wrong 😅